lib/build/index.js

```diff
@@ -47,7 +47,12 @@ exports.bundle = function() {
 
 exports.dest = function() {
   return bundler.build()
-    .then(() => bundler.write());
+    .then(() => bundler.write())
+    .then(() => bundler.copyFiles())
```

Comment:
Maybe make the copyFiles function part of the write command for each bundle, that way you only copy the files when you're doing a write for that particular bundle...